pkgs/applications/misc/kjv/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "kjv";
+  version = "20190311";
+
+  src = fetchFromGitHub {
+    owner = "LukeSmithxyz";
+    repo = "${name}";
+    rev = "f6e6f639b135414659179b09a5fdec4ad8add40e";
+    sha256 = "1yn91v8rsfdsd8p1bfx3v54cvyzwg5mka3inzdkc60k8dahfq8g5";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "The Bible, King James Version";
+    homepage = "https://github.com/LukeSmithxyz/${name}";
```

Comment:
try this instead

```suggestion
    inherit (src.meta) homepage;
```

Reply:
although better on the DRY principle, there's a lot of cases when im fixing package that i just want to be able to copy and paste where the thing is located.

Reply:
check out this:

```bash
nix-homepage () { nix-instantiate --eval --expr "with import <nixpkgs> {}; pkgs.$1.meta.homepage" | sed 's,^",,;s,"$,,' }
```

Reply:
still seems quite removed from the convenience and explicitness of a url located with the other meta information.

I'm not saying one is strictly better, I have a personal leaning toward to a raw url or string.

Reply:
yeah I'm cool with either, I don't have a strong opinion on it.